### PR TITLE
Post-merge suggestions of resource management

### PR DESCRIPTION
**Pull request by Raalsky**
Applied post-merge suggestions to https://github.com/neptune-ai/neptune-client/pull/1609

## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

**Comment by Raalsky** [comment]
E2E: https://github.com/neptune-ai/neptune-client/actions/runs/7787523547

**Review by szysad — APPROVED** [review]

File: src/neptune/cli/status.py [hunk 1]
```diff
@@ -62,8 +62,11 @@ def status(*, backend: "NeptuneBackend", path: Path) -> None:
                 "\nWARNING: %s objects was skipped because they do not exist anymore.",
                 len(containers.not_found_containers),
             )
-
-        if not any([containers.synced_containers, containers.unsynced_containers, containers.offline_containers]):
+        if (
+            not containers.synced_containers
+            and not containers.unsynced_containers
+            and not containers.offline_containers
+        ):
```
[the remark this answers is not in the document]
Comment (Raalsky) [reply src/neptune/cli/status.py:69]: Ok, so let me reapply his suggestion :wink:

[the remark this answers is not in the document]
Comment (Raalsky) [reply src/neptune/cli/status.py]: Done

**Review by AleksanderWWW — APPROVED** [review]
Approving so that we can release a candidate today :white_check_mark:

**Pull request merged**